product_docs/docs/pgd/5/manual/04-installing-software.mdx

```diff
@@ -0,0 +1,249 @@
+---
+title: Installing software
+deepToC: true
+---
+
+With the repositories configured, you can now install the Postgres Distributed software. 
+These steps must be carried out on each host.
+
+We will detail the steps for EDB Postgres Advanced Server below. If you are installing PGD with EDB Postgres Extended Server or Community Postgres, the steps are similar, but details such as package names and paths are different. These differences are summarized in [Installing PGD for EDB Postgres Extended Server](#installing_pgd_for_edb_postgres_extended_server) and [Installing PGD for Community Postgresql](#installing_pgd_for_community_postgresql).
+
+## Installing PGD for EDB Postgres Advanced Server
+
+### Install the packages
+
+The first step is to install the packages. For each Postgres package, there is a `edb-bdr5-<postgresversion>` package to go with it. 
+In our example, we are installing EDB Postgres Advanced Server (epas) version 16, and so will install `edb-bdr5-epas16`. 
+
+There are two other packages to also install: `edb-pgd5-proxy` for PGD Proxy and `edb-pgd5-cli` for the PGD command line tool.
+
+To install all of these packages on a RHEL or RHEL compatible Linux, run:
+
+```
+sudo dnf -y install edb-bdr5-epas16 edb-pgd5-proxy edb-pgd5-cli
+```
+
+### Initializing and starting the database
+
+Now initialise the database. You will need to know the path to the setup script for your particular Postgres. 
+For EDB Postgres Advanced Server, this script can be found in `/usr/edb/as16/bin` as `edb-as-16-setup`. 
+This command needs to be run with the `initdb` parameter and we need to pass an option setting the database to use UTF-8.
+
+
+```
+sudo PGSETUP_INITDB_OPTIONS="-E UTF-8" /usr/edb/as16/bin/edb-as-16-setup initdb
+```
+
+Once the database is initialised, we will start it which will enable us to continue configuring the BDR extension.
+
+```
+sudo systemctl start edb-as-16
+```
+
+### Configuring the BDR extension
+
+Installing EDB Postgres Advanced Server creates a system user `enterprisedb` with admin capabilities when connected to the database.
+In the following steps, we use the sudo command to take on the role of this user and then run the psql command to connect to the default `edb` database:
+
+```
+sudo -u enterprisedb psql edb -c <SQL COMMAND>
+```
+
+#### Preload the BDR library
+
+We want the bdr library to be preloaded with other libraries. 
+EPAS has a number of libraries already preloaded, so we have to prefix the existing list the the BDR library.
+
+```
+sudo -u enterprisedb psql edb -c "ALTER SYSTEM SET shared_preload_libraries TO '\$libdir/bdr','\$libdir/dbms_pipe','\$libdir/edb_gen','\$libdir/dbms_aq'"
```

Comment:
Even though technically correct, we usually are against ALTER SYSTEM usage for these things. 
We suggest appending the values at the end of the postgresql.conf. 
ALTER SYSTEM is indicated for temporary changes.

Reply:
That bumps up the complexity of the guide by quite a bit as we then have to talk through the edits.

ALTER SYSTEM is persistent over multiple runs - only ALTER SYSTEM RESET/RESET ALL makes them disappear. 

But....

We could simply append the content of `postgresql.auto.conf` to `postgresql.conf` and remove the latter to make it permanant....


cat postgresql.auto.conf >> postgresql.conf || rm postgresql.auto.conf

Reply:
I don't like how we suggest this to the customers.
Again, technically, it is ok, but... 
Maybe we can only use `echo` instead of the `ALTER SYSTEM` , directly to the postgresql.conf adn don't bother with ALTER or removing the auto conf....

Reply:
Now reworked with echo ... | sudo tee -a ...